mis_builder/models/aep.py

```diff
@@ -281,14 +284,20 @@ def do_queries(self, date_from, date_to,
                                         ['debit', 'credit', 'account_id'],
                                         ['account_id'])
             for acc in accs:
+                company = account_model.browse(acc['account_id'][0]).company_id
```

Comment:
What do you think about the precision rounding in float_is_zero method ? should it be company currency rounding or AEP currency rounding?